raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -8,18 +8,30 @@ contract NettingChannelContract {
     using NettingChannelLibrary for NettingChannelLibrary.Data;
     NettingChannelLibrary.Data public data;
 
-    event ChannelNewBalance(address token_address, address participant, uint balance);
-    event ChannelClosed(address closing_address);
+    event ChannelNewBalance(
+        address registry_address,
```

Comment:
I think this is non ideal but fine for the moment, since these smart contracts will be replaces by the ones in the https://github.com/raiden-network/raiden-contracts/ repo

Reply:
We should let @loredanacirstea know that the `TokenNetwork` contract should also be initialized with the registry address if we want to do this.

Reply:
the `TokenNetwork` contract already receives the `secret_registry_address` from the `TokenNetworkRegistry` when the contract is created - if this is what you mean.

Reply:
@loredanacirstea that's not it. This version of the smart contracts don't have the secret registry, the `registry_address` here is equivalent to the `token_network_registry_address`

Reply:
edited my comment above as per our discussion